File: Day 26/OutaTimeObjC/OutaTimeObjC/DatePickerViewController.m
//
//  DatePickerViewController.m
//  OutaTimeObjC
//
//  Created by Sean Calkins on 3/7/16.
//  Copyright © 2016 Sean Calkins. All rights reserved.
//

#import "DatePickerViewController.h"

@interface DatePickerViewController ()

@end

@implementation DatePickerViewController

- (void)viewDidLoad {
    [super viewDidLoad];
   
}




- (IBAction)datePickerAction:(UIDatePicker *)sender {
    NSDateFormatter *dateFormatter = [[NSDateFormatter alloc] init];
    [dateFormatter setDateFormat:@"MMM-dd-yyyy"];
    NSString *dateFromPicker = [dateFormatter stringFromDate:self.datePicker.date];
    destinationDate = [dateFromPicker uppercaseString];
    NSLog(@"%@", destinationDate);
    
    
}
@end

File: Day 5/OutaTime/OutaTime/DatePickerViewController.swift
//
//  DatePickerViewController.swift
//  OutaTime
//
//  Created by Sean Calkins on 2/4/16.
//  Copyright © 2016 Sean Calkins. All rights reserved.
//

import UIKit

class DatePickerViewController: UIViewController {

    @IBOutlet weak var datePicker: UIDatePicker!
    
    var destinationDate = ""
    
    @IBAction func datePickerAction(sender: UIDatePicker) {
        let dateFormatter = NSDateFormatter()
        dateFormatter.dateFormat = "MMM-dd-yyyy"
        destinationDate = dateFormatter.stringFromDate(datePicker.date)
        print(destinationDate.uppercaseString)
        
    }
    


}
